### PR TITLE
Add help text to Episode Form

### PR DESCRIPTION
**Pull request by brandonhundt**
- Adds help text throughout the episode form
- Includes some minor styling adjustments to episodes

### CODE REVIEW

File: app/views/episodes/_form.html.erb [hunk 1]
```diff
@@ -1,16 +1,14 @@
 <%= form_with(model: episode, html: {autocomplete: "off"}, data: {controller: "unsaved", unsaved_changed_value: episode.changed?, unsaved_confirm_value: t(".confirm")}) do |form| %>
-  <div class="row mt-4">
+  <div class="row my-4">
     <div class="col-lg-8">
-      <div class="row">
```
Comment (brandonhundt) [inline app/views/episodes/_form.html.erb:4]: This row was messing with the spacing. bye.

File: app/views/episodes/_form_status.html.erb [hunk 2]
```diff
@@ -24,32 +24,34 @@
     </div>
   </div>
 
-  <div class="card-footer">
+  <div class="card-footer d-flex align-items-center justify-content-between">
```
Comment (brandonhundt) [inline app/views/episodes/_form_status.html.erb:27]: This ensures the buttons all remain in alignment and properly spaced with the status text.

File: app/views/episodes/_form_status.html.erb [hunk 3]
```diff
@@ -24,32 +24,34 @@
...
     <% if episode.persisted? %>
       <p class="status-text"><strong><%= t(".updated_at_hint") %></strong> <br><%= l(episode.updated_at) %></p>
 
       <% if policy(episode).update? %>
-        <%= link_to t(".discard"), edit_episode_path(episode), class: "btn btn-discard-changed invisible me-2", data: {action: "unsaved#discard", unsaved_target: "discard"} %>
+        <div>
+          <%= link_to t(".discard"), edit_episode_path(episode), class: "btn btn-danger btn-discard-changed invisible", data: {action: "unsaved#discard", unsaved_target: "discard"} %>
```
Comment (brandonhundt) [inline app/views/episodes/_form_status.html.erb:33]: add some styling to the discard button

**Review by cavis — APPROVED** [review]
👍

**Pull request merged**